<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLAM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>2</y>
      <width>447</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="displayLayout">
     <item>
      <widget class="QLabel" name="speedLabel">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="speedLcdNumber"/>
     </item>
     <item>
      <widget class="QLabel" name="FPSLabel">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="memLabel">
       <property name="text">
        <string>Memory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="memLcdNumber"/>
     </item>
     <item>
      <widget class="QLabel" name="MBLabel">
       <property name="text">
        <string>MB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frameLabel">
       <property name="text">
        <string>Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="frameLcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>519</y>
      <width>269</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="controlLayout">
     <item>
      <widget class="QPushButton" name="stepButton">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Motion"/>
    <addaction name="actionSave_Map"/>
    <addaction name="actionSave_Tracks"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_Map">
   <property name="text">
    <string>Save Map</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionSave_Motion">
   <property name="text">
    <string>Save Motion</string>
   </property>
  </action>
  <action name="actionSave_Tracks">
   <property name="text">
    <string>Save Tracks</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
